java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S2390.html

```diff
@@ -23,7 +26,7 @@ <h2>Resources</h2>
 <ul>
   <li> <a href="https://www.securecoding.cert.org/confluence/display/java/DCL00-J.+Prevent+class+initialization+cycles">CERT, DCL00-J.</a> - Prevent
   class initialization cycles </li>
-  <li> Java Language Specifications - <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-12.html#jls-12.4">Section 12.4: Initialization of
-  Classes and Interfaces</a> </li>
+  <li> <a href="https://docs.oracle.com/javase/specs/jls/se17/html/jls-12.html#jls-12.4">Section 12.4: Initialization of Classes and Interfaces</a> -
+  Java Language Specifications </li>
```

Comment:
The JLS is a single specification
```suggestion
  Java Language Specification </li>
```